ert3/console/_console.py

```diff
@@ -78,6 +96,44 @@ def _build_argparser():
     return parser
 
 
+def _get_ert3_examples_path():
+    pkg_examples_path = pathlib.Path(pkg.resource_filename("ert", "../ert3_examples"))
```

Comment:
Would it not be more correct to ask for `pkg.resorce_filename("ert3_examples")`?

Reply:
`resource_filename` needs two arguments and it wouldn’t work with just `ert3_examples`. However, I do agree that it looks weird, maybe I am misusing function `resource_filename` and it should be done differently

Reply:
I think what is weird is that we are using the location of `ert` to locate `ert3_examples`. And this is not necessarily correct, although it is as installation is set up today...

I think `pkg.resource_filename("ert3_examples", ".")` is better and should do the same for you 🤔